lib/puppet/pops/evaluator/compare_operator.rb

```diff
@@ -59,6 +59,8 @@ def equals_String(a, b)
   def cmp_Numeric(a, b)
     if b.is_a?(Numeric)
       a <=> b
+    elsif b.is_a?(Time::Timespan) || b.is_a?(Time::Timestamp)
+      [0, -1, 1][b <=> a] # compare other way and invert result
```

Comment:
Looks like a somewhat cumbersome way to switch sign. Why not `-(b <=> a)` ?

Reply:
:-) will change